index.html

```diff
@@ -170,6 +170,10 @@
   </div>
 </div>
 
+<p>
```

Comment:
This paragraph needs to be inside those two `div` tags. I'm just gonna move this guy up a few lines and then we'll be good to go!